src/ra.erl

```diff
@@ -374,6 +375,12 @@ start_cluster(System, ClusterName, Machine, ServerIds, Timeout)
 %% If a cluster could not be formed any servers that did manage to start are
 %% forcefully deleted.
 %% @end
```

Comment:
Should this be documented as deprecated? Other functions that use the default `SystemId` are marked as deprecated like `start_server/1`/`restart_server/1`/`stop_server/1`

Reply:
Then it would not make much sense to accept this PR, would it?

Reply:
I think the change is ok as a fix since it restores backwards compatibility. But if we plan on removing the functions that don't specify `SystemId` in the long run, maybe we shouldn't accept this, eh?

Reply:
Let's accept this and deprecate